packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,397 @@
+                            handleChangeReceiverId={receiverId => setReceiverId(receiverId)}
+                            checkAccountAvailable={accountId => dispatch(checkAccountAvailable(accountId))}
+                            localAlert={localAlert}
+                            autoFocus={!isMobile()}
+                            clearLocalAlert={() => dispatch(clearLocalAlert())}
+                            setAccountIdIsValid={setAccountIdIsValid}
+                        />
+                    </div>
+
+                    <ModalFooter>
+                        <div className='buttons'>
+                            <FormButton
+                                className='link'
+                                type='button'
+                                onClick={onClose}
+                                color='gray'
+                            >
+                            <Translate id='NFTTransfer.cancel'/>
+                            </FormButton>
+                            <FormButton
+                                className='next-btn'
+                                type='submit'
+                                disabled={!accountIdIsValid}
+                                onClick={() => setViewType('confirm')}
+                            >
+                            <Translate id='NFTTransfer.next'/>
+                            </FormButton>
+                        </div>
+                    </ModalFooter>
+                </form>
+            </StyledContainer>
+            }
+
+            {viewType === 'confirm' &&
+                <StyledContainer className='small-centered'>
+                    <h3><Translate id='NFTTransfer.transferNft'/></h3>
+
+                    <div className='confirm-nft-card'>
+                        <div className='confirm-img'>
+                            <img src={nft.metadata.mediaUrl} alt='NFT'/>
+                        </div>
+
+                        <div className='line'></div>
+                        <div className='from-box'>
+                            <span className='confirm-txt v-center'><Translate id='transfer.from'/></span>
+                            <span className='h-right v-center'>
+                                <span className='account-id'>{accountId}</span>
+                                <Balance amount={nearBalance} showBalanceInUSD={false}/>
+                            </span>
+                        </div>
+                        <div className='line'></div>
+                        <div className='to-box'>
+                            <span className='confirm-txt v-center'><Translate id='transfer.to' /></span>
+                            <span className='h-right v-center account-id'>{receiverId}</span>
+                        </div>
+                    </div>
+
+                    <div className='estimate-fee-card'>
+                        <EstimatedFees gasFeeAmount={NFT_TRANSFER_GAS}/>
+                    </div>
```

Comment:
```suggestion
                    <EstimatedFees gasFeeAmount={NFT_TRANSFER_GAS}/>
```

No need to wrap and style :)